tools/eclipse-temurin/19-jre-jammy/build.gradle.kts

```
@@ -0,0 +1,10 @@
+apply(plugin = "com.palantir.docker")
+
+tasks {
+    configure<com.palantir.gradle.docker.DockerExtension> {
+        name = "ghcr.io/ministryofjustice/hmpps-probation-integration-services/eclipse-temurin:19-jre-jammy"
+        tag("19-jre-jammy", "ghcr.io/ministryofjustice/hmpps-probation-integration-services/eclipse-temurin:19-jre-jammy")
+        buildx = true
+        platform = arrayOf("linux/arm64", "linux/amd64")
```

```suggestion
        buildx(true)
        platform("linux/arm64", "linux/amd64")
```
The Gradle Kotlin DSL isn't very intuitive, but this should work 😄